### PR TITLE
Added tooltips and standardised web/csv columns

### PR DESCRIPTION
**Pull request by Levia**
- Standardised web/csv columns for Project Instance and Installation lists
- Added notes tooltip in the lists

### CODE REVIEW

**Comment by agnessa** [comment]
wow those are some dramatic tooltips! :) not sure if it is just me, but perhaps a more neutral bootstrappy tooltip might fit better with the design? E.g. http://getbootstrap.com/javascript/#tooltips or http://getbootstrap.com/javascript/#popovers (we probably don't need any extra js)

**Comment by agnessa** [comment]
Can we format the created at / updated at timestamps to just YYYY-MM-DD HH:MM

**Comment by agnessa** [comment]
Let's shorten "Number of installations" to "# installations" to make the display more compact

**Comment by Levia** [comment]
I used those tooltips because they are easier to customise, especially when the description is very long :D . I'm going to try and change those then

**Comment by Levia** [comment]
Tooltips should be ok now

File: app/models/project_instance.rb [hunk 1]
```diff
@@ -33,6 +35,8 @@ def self.to_csv(options = {})
     end
   end
 
-  accepts_nested_attributes_for :comments, :reject_if => lambda { |a| a[:content].blank? }
+  def info
```
Comment (agnessa) [inline app/models/project_instance.rb]: I think this is a good fit for a view helper - the models should have nothing to do with html in an ideal world

**Pull request merged**